.github/actions/e2e-tests/action.yml

```diff
@@ -33,12 +48,10 @@ runs:
           STATIC_ASSET_CDN_URL: /
           ENV: ${{ inputs.environment }}
           ENABLE_INFLUXDB_FEATURES: ${{ inputs.enable_influxdb_features }}
+
         run: |
-          wget -q https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_96.0.4664.110-1_amd64.deb
```

Comment:
Don't we need a specific version of Chrome?

Reply:
Nah, its actually better to use latest stable